src/store/reducers/index.ts

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from '@reduxjs/toolkit'
```

Comment:
Rename this file to `store/reducer.ts` will be okay, we don't need the `reducers` folder anyway.